GMnetENGINE.gmx/scripts/htme_disconnectNow.gml

```diff
@@ -0,0 +1,37 @@
+///htme_disconnectNow()
+
+/*
+**  Description:
+**      Disconnect and clean client or server when game started
+**      This is very handy, just call it and if return true then 
+**      clean your ds maps and lists and go to another room
+**      This will restart the engine, ready to to whatever you want.
+**      Like connect again, start server, lobby...
+**      Like a game restart without the game_restart()
+**  
+**  Usage:
+**      <see above>
+**
+**  Arguments:
+**      <none>
+**
+**  Returns:
+**      true, if ok disconnect, false if not.
+**
+*/
+if instance_exists(global.htme_object) {
```

Comment:
New way to disconnect server or client. Just call it. If it returns true you can go to another room.